client/my-sites/plan-features/footer.jsx

```diff
@@ -4,25 +4,71 @@
 import { localize } from 'i18n-calypso';
 import noop from 'lodash/noop';
 import React, { PropTypes } from 'react';
+import filter from 'lodash/filter';
+import reduce from 'lodash/reduce';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import Gridicon from 'components/gridicon';
+import classNames from 'classnames';
+import { plansList } from 'lib/plans/constants';
 
-const PlanFeaturesFooter = ( { translate, current = false, available = true, description, onUpgradeClick = noop } ) => {
+function getLongestDescription( planName ) {
+	if ( ! planName ) {
+		return null;
+	}
+	const isJetpack = planName.indexOf( 'jetpack' ) === 0;
+	const plans = Object.keys( plansList );
+
+	return reduce(
+		filter( plans, ( plan ) => {
+			const isPlanJetpack = plan.indexOf( 'jetpack' ) === 0;
```

Comment:
This is not very clear. What's the difference between `isJetpack` and `isPlanJetpack`? I would name it as follows:

`isJetpack` -> `isSelectedJetpackPlan`
`isPlanJetpack` -> `isJetpackPlan`

Or something like that which suggests that one variable is for the current plan and one for the looped plans.